mirrord/layer/src/load.rs

```diff
@@ -0,0 +1,84 @@
+use mirrord_config::{util::VecOrSingle, LayerConfig};
+
+/// For processes that spawn other processes whe should also patch sip on the spawned instances,
+/// curretly limiting to list from `SIP_ONLY_PROCESSES`
+#[cfg(target_os = "macos")]
```

Comment:
Add a comment explaining that this list is being hit only when it's also in the `skip processes` list